app/src/main/res/values/strings.xml

```diff
@@ -5,24 +5,35 @@
   -->
 
 <resources xmlns:tools="http://schemas.android.com/tools">
+    <!-- This string is the full and formal title of the application, used in areas like a header or welcome screen -->
     <string name="app_name">Firefox Lockbox</string>
+    <!-- This is the short name or label of the application, used as the name in the Android app launcher -->
     <string name="app_label">Lockbox</string>
+    <!-- This string is a promotional or short descriptional sentence to describe the purpose of the application -->
     <string name="app_tagline">Take your passwords everywhere.</string>
 
+    <!-- This string is used as the label on a button on a first-run welcome screen to sign in and start using the application -->
     <string name="welcome_start_btn">Get Started</string>
+    <!-- This string provides context near the "welcome_start_btn" to describe what using the application will require -->
     <string name="welcome_instructions">To use Firefox Lockbox, you\'ll need a\nFirefox Account with saved logins.</string>
```

Comment:
Can we pass in the brandname here? If not, the localization note should be explicit about not allowing to modify `Firefox Lockbox`. Similar for the other strings containing the lockbox brand.

Also, should we use proper curly apostrophe in our copy on Lockbox? We do in all/most Firefox products.

Reply:
+1 to using curly apostrophe.

Concerning the brand name: can't we just use something like %1$s which we use for other mobile Android product names?